dashboard/src/module/purchase-credit/index.tsx

```diff
@@ -461,7 +461,7 @@ const BuyCreditsCard = ({ token }: { token?: string }) => {
       api.setSigner(injector.signer);
 
       const transfer = api.tx.balances.transferKeepAlive(
-        selected?.address as `0x${string}`,
+        process.env.NEXT_PUBLIC_ADDRESS as string,
```

Comment:
add a new env variable called "NEXT_AVAIL_PUBLIC_ADDRESS". Due to bad naming in the code atm this might seem like it will work, but this is actually the deposit smart contract address on ethereum and other rollups ( detereministic address, same on all supported chains )